holidayextras/static-site-generator
# Upgrade Babel to v6.X

#### What does this PR do? (please provide any background)

Upgrades the babel version to the latest stable release.
#### What tests does this PR have?

None
#### How can this be tested?

The best way to test this is to run `npm run trans` that should babelify the static sit generator. If succesful we are all good. 
#### What domains are affected?

None - This is just the generator
#### What gif best describes how you feel about this work?

![OH YEA](https://media.giphy.com/media/BUUHYFWW8BIOs/giphy.gif)

---

**Notes when reviewing Static Site Generator projects, please think carefully about what can be done to centralise common functionality. Please also pay attention to anything configurable that will also need updating in the Yeoman generator template**
#### Reviewers

**Review 1**

**Review 2**

**Review 3**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.
